thunder/executors/nvfuserex_impl.py

```diff
@@ -442,6 +442,10 @@ class FusionDefinitionWrapper:
     last_used: None | FusionDefinition = None
     last_inputs: None | Sequence[tuple] = None
     store_inputs: bool = False
+    print_repro: bool = False
+    # Pre-generated so that we don't construct a LooseVersion on the critical
+    # path.
+    repro_version_needed = LooseVersion("0.2.22")
```

Comment:
nitpick: Does this need to be a per instance field, rather than a file scope thing?

Reply:
Good point! Thanks.